bin/dfx-canister-url.test

```diff
@@ -112,12 +112,13 @@ print_title() {
     echo "ERROR in test setup:  Failed to set canister ID."
   }
   : "Compare with the actual URL"
-  export ACTUAL_POSITIONAL_URL ACTUAL_FLAG_URL ACTUAL_POSITIONAL_URL_WITH_EXPLICIT_NETWORK ACTUAL_FLAG_URL_WITH_EXPLICIT_NETWORK
+  export ACTUAL_POSITIONAL_URL ACTUAL_FLAG_URL ACTUAL_POSITIONAL_URL_WITH_EXPLICIT_NETWORK ACTUAL_FLAG_URL_WITH_EXPLICIT_NETWORK ACTUAL_POSITIONAL_URL_WITH_RAW
   ACTUAL_POSITIONAL_URL="$(dfx-canister-url smiley_dapp_assets)"
   ACTUAL_FLAG_URL="$(dfx-canister-url --canister smiley_dapp_assets)"
   ACTUAL_POSITIONAL_URL_WITH_EXPLICIT_NETWORK="$(dfx-canister-url --network local smiley_dapp_assets)"
   ACTUAL_FLAG_URL_WITH_EXPLICIT_NETWORK="$(dfx-canister-url --network local --canister smiley_dapp_assets)"
-  for actual_varname in ACTUAL_POSITIONAL_URL ACTUAL_FLAG_URL ACTUAL_POSITIONAL_URL_WITH_EXPLICIT_NETWORK ACTUAL_FLAG_URL_WITH_EXPLICIT_NETWORK; do
+  ACTUAL_POSITIONAL_URL_WITH_RAW="$(dfx-canister-url --raw smiley_dapp_assets)"
+  for actual_varname in ACTUAL_POSITIONAL_URL ACTUAL_FLAG_URL ACTUAL_POSITIONAL_URL_WITH_EXPLICIT_NETWORK ACTUAL_FLAG_URL_WITH_EXPLICIT_NETWORK ACTUAL_POSITIONAL_URL_WITH_RAW; do
     [[ "$EXPECTED_URL" == "${!actual_varname}" ]] || {
```

Comment:
I know this logic was here before but I'm confused. How can all of these have the same expected URL?
I'm thinking a long list of explicit calls with explicit expected URLs might be easier to follow.

Reply:
This comment was clobbered but I can answer here.

> How can all of these have the same expected URL?

Because these commands are all asking for the same thing.  Or can you see two commands that should be giving different outputs?